system/jlib/jlzw.cpp

```diff
@@ -1985,10 +1997,13 @@ class CCompressedFile : implements ICompressedFileIO, public CInterface
     MemoryAttr prevrowbuf; 
     bool setcrc;
     bool writeException;
+    bool keepInputBlock = true; // depends on new expansion code
```

Comment:
when would you not want this to be true?

Reply:
Compression that is not incremental e.g. lzw.  I think it makes sense to always be true though because it saves lots of allocation and reallocation.